parm/config/gfs/config.verfozn

```diff
@@ -6,4 +6,19 @@ echo "BEGIN: config.verfozn"
 # Get task specific resources
 . "${EXPDIR}/config.resources" verfozn
 
+export DO_DATA_RPT=1
+export OZN_AREA="glb"
+export OZNMON_SUFFIX=${NET}
+export PARMmonitor=${PARMgfs}/monitor
+export SATYPE_FILE=${PARMmonitor}/gdas_oznmon_satype.txt
+
+# Source the parm file
+export parm_file=${PARMmonitor}/gdas_oznmon.parm
+. "${parm_file}"
```

Comment:
Is it necessary to both source the file and export the path to the file? And is the file coming from elsewhere, or should those settings also be added here?

Reply:
I can compress those two lines. That parm file comes from the GSI-Monitor package under `/sorc` and gets symlinked during setup. It has settings we could pull into the config and then ignore the parm file from the package:
```
kate.friedman@clogin09:/lfs/h2/emc/global/noscrub/kate.friedman/git/feature-combine_vminmon_scripts> cat parm/monitor/gdas_oznmon.parm
##################################
# specify oznmon utilities
##################################
export COMPRESS=${COMPRESS:-gzip}
export UNCOMPRESS=${UNCOMPRESS:-gunzip}
export VALIDATE_DATA=${VALIDATE_DATA:-1}
export Z=${Z:-gz}
kate.friedman@clogin09:/lfs/h2/emc/global/noscrub/kate.friedman/git/feature-combine_vminmon_scripts> ll parm/monitor/gdas_oznmon.parm
lrwxrwxrwx 1 kate.friedman global 155 Nov 28 13:42 parm/monitor/gdas_oznmon.parm -> /lfs/h2/emc/global/noscrub/kate.friedman/git/feature-combine_vminmon_scripts/sorc/gsi_m
onitor.fd/src/Ozone_Monitor/nwprod/gdas_oznmon/parm/gdas_oznmon.parm
```